scripts/measures/2018/import-qcdr-measures.js

```diff
@@ -79,10 +79,17 @@ const config = {
     primarySteward: 20
     // `metricType` is a sourced field but not represented here since it maps from
     // multiple columns-- you can find it by searching in the code below
+  },
+  special_fields: {
```

Comment:
what do you think of calling this `referenced_fields`, if that accurately captures it, and changing sourced_fields to `mapped_fields`? also, can you add a new comment below `'source_fields' are fields which...` at the top of this file, describing `referenced_fields` (or whatever it's named)?